handlers/handler.go

```diff
@@ -135,3 +135,24 @@ func IndexS3Object(s3objectURL string) {
 
 	updateMetadataObjectWrapper(uuid, configInfo, mdsUploadedBody)
 }
+
+func CreateUUID(key string) (string, string, error) {
+	split_key := strings.Split(key, "/")
+	var uuid string
+	var filename string
+	// here uuid can be dg.XXXX/<uuid> or
+	// just <uuid> to distinguish between
+	// them and to keep the filename consistent
+	foundGUID, errGUID := id.Parse(split_key[0])
+	foundPrefix, errPrefix := id.Parse(split_key[1])
```

Comment:
Is this going to be a problem when there is no prefix? Index 1 of `split_key` is referenced, but what if there was no `"/"` in `key`?

Reply:
added logic to prevent this